variables.tf

```
@@ -0,0 +1,30 @@
+
+variable "tags" {
+  type        = map(string)
+  default     = {}
+  description = "Additional tags. Overwrite module-generated tags"
+}
+
+variable "timestamp_format" {
+  type        = string
+  default     = "YYYY-MM-DD'T'hh:mm:ssZ"
+  description = "Timestamp format, see terraform.io/docs/configuration/functions/formatdate.html#specification-syntax"
+}
+
+variable "enforce_case" {
+  type        = string
+  default     = "None"
+  description = "Type casing to enforce. Options are: [ None (default) | UPPER | lower ]"
+}
+
+variable "names" {
+  type        = set(string)
```

This should be a list, not a set. Maintaining the order is more important than making sure they are unique (sets are unordered)
```suggestion
  type        = list(string)
```